templates/default/etc/init.d/node-ff.erb

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+#
+# chkconfig: - 80 20
+# processname: node-ff
+# description: Selenium Grid node for Firefox
+#
+
+### BEGIN INIT INFO
+# Provides: Selenium Grid Firefox node
+# Required-Start: $network $syslog
+# Required-Stop: $network $syslog
+# Default-Start:
+# Default-Stop:
+### END INIT INFO
+
+PATH_TO_JAR=<%= node['aet']['seleniumgrid']['node_ff']['root_dir'] %>/selenium-server-standalone-3.8.1.jar
+HUB_URL="http://localhost:4444"
+CAPABILITIES="browserName=firefox,version=ANY,platform=LINUX,maxInstances=5"
```

Comment:
Lets remove the `platform` from here as it will be detected by Selenium automatically.